correlations-matrix/readme.md

```diff
@@ -0,0 +1,18 @@
+# Correlation matrix CSV
+
+In this package you'll find a script that generates as output a CSV
+that contains a matrix of correlation values for the fields in a dataset.
+The index used for the
+correlation analysis is the `eta squared` (One-Way ANOVA). Given that this
+correlation can be computed for numeric and categorical fields, only
+the fields that fall into these types are used.
+
+The **inputs** for the script are:
+
+* `dataset-id`: (dataset-id) Dataset ID for the original data
+* `fields`: (list) List of fields to be analyzed (default: [])
+* `options`:  (map) Arguments for the correlation creation (default: {})
+* `correlation-index`: (string) Index to be used (default: "eta_square")
```

Comment:
I asked the backend team about that, and the only correlation index that they suggested was eta_square. However, they mentioned that other indexes could be easily added to the correlation resource that could be also used in this kind of matrices. That's why I offered that index as a parameter, just in case we ended up offering other indexes.